tests/queries/0_stateless/02477_projection_materialize_and_zero_copy.sql

```diff
@@ -13,6 +13,6 @@ alter table t materialize projection p_norm settings mutations_sync = 1;
 
 SYSTEM FLUSH LOGS;
 
-SELECT * FROM system.text_log WHERE event_time >= now() - 30 and level == 'Error' and message like '%BAD_DATA_PART_NAME%';
+SELECT * FROM system.text_log WHERE event_time >= now() - 30 and level == 'Error' and message like '%BAD_DATA_PART_NAME%'and message like '%p_norm%';
 
 DROP TABLE IF EXISTS t;
```

Comment:
This test is now failing, see for example https://s3.amazonaws.com/clickhouse-test-reports/43104/ae19af00156586acb5d1af09d4564f957ed06220/fast_test.html

```
Error 85f2827d-b501-44cb-8dd7-6af299f58a34 TCPHandler Code: 233. DB::Exception: Detached part "all_4_4_0" not found. (BAD_DATA_PART_NAME)
```